nekoyume/broadcast.py

```diff
@@ -0,0 +1,137 @@
+import datetime
+import os
+from typing import Mapping, Optional
+import urllib.parse
+
+from requests import post
+from requests.exceptions import ConnectionError, Timeout
+
+from .block import Block
+from .move import Move
+from .node import Node
+from .orm import db
+
+
+DEFAULT_BROADCAST_LIMIT = os.environ.get('BROADCAST_LIMIT', 100)
```

Comment:
Although the name implies it's a constant value only for lack of configuration (i.e., default), it's actually used every situation.  So I suggest one of the following either ways:

- Remove the prefix `DEFAULT_`, or

- Let `broadcast_block()` to take `limit` argument, e.g.:

    ```python
    def broadcast_block(
            serialized: Mapping[str, object],
            sent_node: Optional[Node]=None,
            my_node: Optional[Node]=None,
            limit: Optional[int]=DEFAULT_BROADCAST_LIMIT,
    ):
    ```